lib/puppet/util/windows/taskscheduler.rb

```diff
@@ -809,7 +809,7 @@ def max_run_time=(max_run_time)
     # Returns whether or not the scheduled task exists.
     def exists?(job_name)
       bool = false
-      Dir.foreach('C:/Windows/Tasks'){ |file|
+      Dir.foreach("#{ENV['windir']}/Tasks"){ |file|
```

Comment:
Here's why http://superuser.com/questions/638321/what-is-difference-between-windir-and-systemroot.